src/utilities/text/text.scss

```diff
@@ -0,0 +1,9 @@
+@use "../../compiled/tokens/scss/color";
+
+.u-textNoWrap {
+  white-space: nowrap !important;
+}
+
+.u-textColorBrandPrimary {
+  color: color.$brand-primary !important;
```

Comment:
How does this work when themes are applied?

Reply:
Great question! Hmm...I'll have to think about this. 🤔